card/lib/card/set/loader.rb

```diff
@@ -52,15 +52,36 @@ def pattern_and_modules_from_path path
       end
 
       def tmp_file_template pattern, modules, content_path
+        content = File.read(content_path)
+        wrapped_content = tmp_file_wrapped_content content_path, content
+
+        # TODO: load directly without tmp file
+        return wrapped_content if content =~ /\A#!\s?simple load/
```

Comment:
I like the distinction between `simple load` and `not set module`.  But is simple load really loaded right away, or is a tmp file created?  Should we just load it?